api/client/lib.py

```diff
@@ -438,7 +438,11 @@ def get_data_series(access_token, api_host, **selection):
     params = get_params_from_selection(**selection)
     resp = get_data(url, headers, params)
     try:
-        return resp.json()['data']
+        response = resp.json()['data']
+        if any(map(lambda series: 
```

Comment:
List comprehension is preferred over maps: `[series.get('historical_region', False) for series in response]`

Reply:
done